orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/servergroup/ServerGroupCacheForceRefreshTask.groovy

```diff
@@ -158,7 +159,11 @@ class ServerGroupCacheForceRefreshTask extends AbstractCloudProviderAwareTask im
         }
 
         def forceCacheUpdate = pendingForceCacheUpdates.find {
-          (it.details as Map<String, String>).intersect(model) == model
+          if (it) {
```

Comment:
Why is this needed?

Reply:
It is possible to send null to here because both agent is send data here here.

Reply:
That seems like an error on the cloud provider agent, I can't think of a good reason why `pendingForceCacheUpdates` would contain a `null` value.

Reply:
I will take a look if it is possible filter in clouddriverside  if null.